lib/http.js

```diff
@@ -0,0 +1,31 @@
+'use babel'
+
+import token from './token'
+
+function httpRequest(method, path, body, learnAuth=false) {
```

Comment:
Yeah, it'd definitely be nice & would save a lot of time to have a standard throughout